hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1620,18 +1620,27 @@
 
   <property>
     <name>ozone.om.ratis.minimum.timeout</name>
-    <value>1s</value>
+    <value>5s</value>
     <tag>OZONE, OM, RATIS, MANAGEMENT</tag>
     <description>The minimum timeout duration for OM's Ratis server rpc.
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.ratis.leader.step.down.wait.time</name>
+    <value>30s</value>
+    <tag>OZONE, OM, RATIS, MANAGEMENT, DEPRECATED</tag>
+    <description>The time OM Ratis Leader waits before stepping down on loosing
+      majority heartbeats from followers.
+    </description>
+  </property>
+
   <property>
     <name>ozone.om.leader.election.minimum.timeout.duration</name>
-    <value>1s</value>
-    <tag>OZONE, OM, RATIS, MANAGEMENT</tag>
-    <description>The minimum timeout duration for OM ratis leader election.
-      Default is 1s.
+    <value>5s</value>
+    <tag>OZONE, OM, RATIS, MANAGEMENT, DEPRECATED</tag>
+    <description>DEPRECATED. Leader election timeout uses ratis
+      rpc timeout which can be set via ozone.om.ratis.minimum.timeout.
```

Comment:
OUt of curiosity:

so the deprecation of a  config entry is to put `DEPRECATED` in the description?

Reply:
I took reference from Hadoop. There is a DeprecationDelta we can add in OzoneConfiguration but since this config is not being replaced by a new config, I was not sure if it makes sense to add it there. 

@adoroszlai, thoughts? should this config be added to OzoneConfiguration#deprecated keys?

Reply:
I don't think we need to add it to deprecated keys. This should not be a widely used config key.